inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -362,13 +362,6 @@ int main(int argc, char* argv[]) {
 
             auto startTime = Time::now();
             CNNNetwork cnnNetwork = ie.ReadNetwork(FLAGS_m);
-            // ngraph::frontend::FrontEndManager manager;
```

Comment:
I propose to remove all redundant comments from benchmark app.

Reply:
I don't see any more redundant comments in benchmark_app